index.html

```diff
@@ -282,9 +282,9 @@ <h5 class="partners-title">Partner</h5>
         <!-- footer section -->
 
         <section class="footer-section container footer-content d-flex justify-content-center">
-                <div class="footer-logo">
+                <h5 class="footer-logo">
                     GCRECC
-                </div>
```

Comment:
- Kindly finish your about section and make sure that it has all the sections that is visible on the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). You currently have only the headline section.

You should be having:
on the Headline:
- Please use the same color for the Heading as in the design which is `#ec5242`. Kindly do something similar for the heading on the `homepage` as well.
About
![image](https://user-images.githubusercontent.com/38088818/169255573-f1743201-bafc-4790-9454-f8a98be42de4.png)
Homepage
![image](https://user-images.githubusercontent.com/38088818/169255669-d492da74-7087-4941-a827-463793b06227.png)

In the second section:
- Kindly add a similar section on your about page that is relevant to your theme.
![image](https://user-images.githubusercontent.com/38088818/169256335-604605fa-e682-4e05-a86b-88e30675204f.png)

In the third section:

- Kindly add past global summits with at least two cards that depict past events.
![image](https://user-images.githubusercontent.com/38088818/169256753-07f34519-8ea8-499a-b5a2-f3c8a37e4fd6.png)

And a footer that is similar to the homepage footer but with a dark background as in the design.
![image](https://user-images.githubusercontent.com/38088818/169256972-5ef439bc-04d7-4b44-96c8-5fe1ffc16f66.png)